focal/Dockerfile

```diff
@@ -6,30 +6,31 @@ SHELL ["/bin/bash", "-c"]
 # Common tools and dependencies, GCC, ASDF
 # Note: We setup locales using the snippet from `ubuntu` image readme.
 RUN apt-get update \
- && apt-get install -y software-properties-common \
- && add-apt-repository ppa:ubuntu-toolchain-r/test -y \
- && apt-get update \
- && apt-get install -y \
-    autoconf \
-    automake \
-    build-essential \
-    curl \
-    gcc-9 \
-    git \
-    libffi-dev \
-    libncurses-dev \
-    libreadline-dev \
-    libssl-dev \
-    libtool \
-    libxslt-dev \
-    libyaml-dev \
-    locales \
-    unixodbc-dev \
-    unzip \
-    wget \
- && rm -rf /var/lib/apt/lists/* \
- && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
- && git clone https://github.com/asdf-vm/asdf.git /root/.asdf -b v0.7.8
+  && apt-get install -y software-properties-common \
+  && add-apt-repository ppa:ubuntu-toolchain-r/test -y \
+  && apt-get update \
+  && apt-get install -y \
+  autoconf \
+  automake \
+  build-essential \
+  curl \
+  gcc-9 \
+  git \
+  libffi-dev \
+  libncurses-dev \
+  libreadline-dev \
+  libssl-dev \
+  libtool \
+  libxslt-dev \
+  libyaml-dev \
+  locales \
+  unixodbc-dev \
+  unzip \
+  wget \
+  clang-format \
```

Comment:
1) seems like formatting got messed up
2) please keep this list sorted :)

Reply:
1. vscode formats like that, if there are any specific formatting rules let's specify them somewhere

Reply:
vscode has very dumb formatting for Dockerfiles, I recommend manual formatting :)

Reply:
done